README.md

```diff
@@ -24,9 +24,9 @@ but here are a few examples of "before"/"after" to whet your appetite:
 ```hbs
 {{site-header user=this.user class=(if this.user.isAdmin "admin")}}
 
-{{#super-select selected=this.user.country as |s|}}
+{{#super-select selected=this.user.country as |option|}}
   {{#each this.availableCountries as |country|}}
-    {{#s.option value=country}}{{country.name}}{{/s.option}}
+    {{#option value=country}}{{country.name}}{{/s.option}}
```

Comment:
You can't call a non-angle bracket component without a dash, that is why the API is different. To make it similar you'd use `{{component option`.

Reply:
Of course. Missed that one. So maybe it should be done the other way round? Assuming it returns a hash and updating the angle bracket syntax accordingly?

Reply:
Or is it attended to show this difference? At least without an explanation that was very confusing to me.

Reply:
The other way around makes sense to me.

Reply:
Updated the PR accordingly.